docs/Basic_Dialogue.md

```diff
@@ -123,6 +123,35 @@ Similarly, if the name of a character is based on a variable you can provide it
 {{SomeGlobal.some_character_name}}: My name was provided by the player.
 ```
 
+### Local variables
+
+If you need temporary variables that only exist during a dialogue conversation, you can use locals. Locals are temporary variables that only live for the current conversation. When the conversation ends or changes dialogue files, the variables are deleted.
```

Comment:
It important to mention that `locals` is implemented in the example balloon (as an example of simple, ephemeral state) and not the Dialogue Manager itself.